File: python/examples/quickstart.py
from twofinance_sdk_client import TwoFinanceClient, config_from_env


def main() -> None:
    client = TwoFinanceClient(config_from_env())

    indicators = client.analytics.indicators()
    print("analytics indicators:", indicators)

    plan = client.planner.trading_plan(
        {
            "goal": "prepare a BTC rebalancing plan",
            "useAnalytics": True,
            "useTrading": True,
        }
    )
    print("planner response:", plan)


if __name__ == "__main__":
    main()

File: typescript/examples/quickstart.ts
import { TwoFinanceClient, configFromEnv } from "../src/index";

async function main() {
  const client = new TwoFinanceClient(configFromEnv(process.env));

  const indicators = await client.analytics.indicators();
  console.log("analytics indicators:", indicators);

  const plan = await client.planner.tradingPlan({
    goal: "prepare a BTC rebalancing plan",
    useAnalytics: true,
    useTrading: true
  });
  console.log("planner response:", plan);
}

main().catch((error) => {
  console.error(error);
  process.exitCode = 1;
});
